js/adapt-vimeo.js

```diff
@@ -114,7 +114,7 @@ class Vimeo extends ComponentView {
     Adapt.trigger('media', {
       isVideo: true,
       type: eventType,
-      src: this.model.get('_media')._source,
+      src: this.model.get('_media')._source.trim(),
```

Comment:
```suggestion
      src: this.model.get('_media')._source?.trim(),
```
You may want to add the optional chaining operator in case the _source object isn't defined